src/main/java/org/jhipster/blog/config/JavaMailSenderConfiguration.java

```diff
@@ -0,0 +1,34 @@
+package org.jhipster.blog.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+
+import java.util.Properties;
+
+@Configuration
+public class JavaMailSenderConfiguration {
+
+    @Bean
+    public JavaMailSender getJavaMailSender() {
+        JavaMailSenderImpl mailSender = new JavaMailSenderImpl();
+        //Your host - example smtp.gmail.com
+        mailSender.setHost("smtp.gmail.com");
```

Comment:
It's better to use properties instead of hardcoded values. If we want to have something for testing, we better create a dedicated profile and put values there.